### DISCUSSION TITLE
@assets ,  @script not working

### DISCUSSION THREAD

**Discussion by nafea107** [livewire/livewire#7333] [1. Help]
@assets ,  @script not working and not rendered properly 
<img src="https://i.ibb.co/Jx5MdZq/Screenshot-from-2023-11-20-19-32-39.png" alt="Screenshot-from-2023-11-20-19-32-39" border="0">

**Comment by parthkapatel** [comment]
Hello ,
 You have to write all the code in one parent div tag per blade file after that this error is resolved

**Comment by nafea107** [comment]
I have done that but @assets not working but @script is working. I'm using @stack and @push instead of @assets but I am wondering why it doesn't work

**Comment by nafea107** [comment]
I have done that but @assets not working but @script is working. I'm using @stack and @push instead of @assets but I am wondering why it doesn't work

**Comment by parthkapatel** [comment]
can you please provide more information with proper code ??

**Comment by nafea107** [comment]
[Screencast from 25 Dec 07:11:56 pm.webm](https://github.com/livewire/livewire/assets/47711093/d976d8cb-4953-464e-b5cc-db49d93fa580)
when accessing page using wire:navigate it does not work but when accessing page as initial page scripts get loaded
